### ISSUE TITLE
http://ec2-54-184-13-84.us-west-2.compute.amazonaws.com: still unreachable

### ISSUE THREAD

**Issue by houseofc** [deepimagination/PoE-GAN#28]
your daily issue for gaugan2 broken

c'mon, it's my birthday tomorrow and i want to play with my favorite ai!

**Comment by 4enetic0** [comment]
No infos or updates statement or similar maybe it was discontinued due to canvas

**Comment by houseofc** [comment]
nope, it is supposed to be shown off at the dev conference in september. could be the maintainer is on vacation or something, or possibly working so hard on 360 that they have no bandwidth for just plain gaugan2
